docs/upgrade/v1-1-2-to-v1-2-0.md

```diff
@@ -398,3 +398,76 @@ If an upgrade is stuck in an `Upgrading System Service` state for an extended pe
   - https://github.com/harvester/harvester/issues/3863#issuecomment-1539681311
 
 ---
+
+### 8. Upgrade stuck in the `Post-draining` state
+
+If you encounter the `post-draining` state like the following figure, that might be stuck in the OS upgrade process.
+
+![](/img/v1.2/upgrade/known_issues/stuck-in-post-draining.png)
+
+Harvester uses the `elemental upgrade` to help us upgrade the OS. Check the `elemental upgrade`` logs to see if there is any error.
+
+You can check the `elemental upgrade` logs with the following command:
+```
+$ kubectl get pod --selector=harvesterhci.io/upgradeJobType=post-drain -n harvester-system
+# you will see the post-drain job, should be named `hvst-upgrade-xxx-post-drain-xxx`
+
+# Then you could check the logs with the following command:
+$ kubectl logs -n harvester-system pods/hvst-upgrade-xxx-post-drain-xxx
+```
+
+Suppose you see the following error in the logs. That will hit the issue caused by incomplete `state.yaml`.
+```
+Flag --directory has been deprecated, 'directory' is deprecated please use 'system' instead
+INFO[2023-09-13T12:02:42Z] Starting elemental version 0.3.1             
+INFO[2023-09-13T12:02:42Z] reading configuration form '/tmp/tmp.N6rn4F6mKM' 
+ERRO[2023-09-13T12:02:42Z] Invalid upgrade command setup undefined state partition 
+elemental upgrade failed with return code: 33
++ ret=33
++ '[' 33 '!=' 0 ']'
++ echo 'elemental upgrade failed with return code: 33'
++ cat /host/usr/local/upgrade_tmp/elemental-upgrade-20230913120242.log
+```
+
+Harvester upgraded the elemental-cli to the latest version. It will try to find the `state` partition from the `state.yaml`. If the `state.yaml` is incomplete, it will have a chance to fail to find the `state` partition.
+
+The incomplete `state.yaml` will look like the following.
+```
+# Autogenerated file by elemental client, do not edit
+
+date: "2023-09-13T08:31:42Z"
+state:
+    # we are missing `label` here.
+    active:
+        source: dir:///tmp/tmp.01deNrXNEC
+        label: COS_ACTIVE
+        fs: ext2
+    passive: null
+```
```

Comment:
```suggestion
The incomplete `state.yaml` will look like the following.

```yaml
# Autogenerated file by elemental client, do not edit

date: "2023-09-13T08:31:42Z"
state:
    # we are missing `label` here.
    active:
        source: dir:///tmp/tmp.01deNrXNEC
        label: COS_ACTIVE
        fs: ext2
    passive: null
```
```